apps/src/templates/readonly.html.ejs

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <title>Blockly</title>
   <link href="<%= assetUrl('css/common.css') %>" media="all" rel="stylesheet">
+  <script type="text/javascript" src="<%= assetUrl('js/manifest.js') %>"></script>
```

Comment:
What if instead of adding manifest.js as a script (which then mysteriously adds a digestManifest to our globals), we have a registerManifest function (possibly a sibling to initApp and setupApp). We pass this a path to our manifest, and it makes an AJAX call and stores the result.